deploy/pull-config/updatescript.js

```diff
@@ -0,0 +1,74 @@
+'use strict';
+
+const https = require('https');
+const fs = require('fs');
+const async = require('./async');
+const cp = require('child_process');
+const report = require('./report').report;
+
+const hardwareCode = process.argv[2];
+
+const clientFolder = hardwareCode === 'aaeon' ? '/var/lib/lamassu-machine' : '/opt/lamassu-machine/data'
```

Comment:
Since `hardwareCode === 'aaeon'` appears multiple times in the file, maybe it would get more clean if you create a variable for it: `const isAaeon = hardwareCode === 'aaeon'`. Do you think it make sense?